src/Features/CSharp/Portable/CodeRefactorings/NodeSelectionHelpers.cs

```diff
@@ -18,8 +19,30 @@ internal static async Task<SyntaxNode> GetSelectedDeclarationOrVariableAsync(Cod
             // MemberDeclaration: member that can be declared in type (those are the ones we can pull up) 
             // VariableDeclaratorSyntax: for fields the MemberDeclaration can actually represent multiple declarations, e.g. `int a = 0, b = 1;`.
             // ..Since the user might want to select & pull up only one of them (e.g. `int a = 0, [|b = 1|];` we also look for closest VariableDeclaratorSyntax.
-            return await context.TryGetRelevantNodeAsync<MemberDeclarationSyntax>().ConfigureAwait(false) as SyntaxNode ??
-                await context.TryGetRelevantNodeAsync<VariableDeclaratorSyntax>().ConfigureAwait(false);
+            var memberDeclaration = await context.TryGetRelevantNodeAsync<MemberDeclarationSyntax>().ConfigureAwait(false) as SyntaxNode;
+            if (memberDeclaration == null)
+            {
+                return await context.TryGetRelevantNodeAsync<VariableDeclaratorSyntax>().ConfigureAwait(false);
+            }
+            // since a FieldDeclarationSyntax doesn't have an associated Symbol (it can have multiple), we just get its variable declaration instead
+            if (memberDeclaration is FieldDeclarationSyntax fieldDeclaration)
+            {
```

Comment:
```suggestion
            if (memberDeclaration is not FieldDeclarationSyntax fieldDeclaration)
                return memberDeclaration;
```

nit.  makes it easier to see what the special logic is doing if the non-special case is simple and handled immediately.